roles/fstat/tasks/main.yml

```diff
@@ -31,3 +31,15 @@
     name: fstat
     enabled: True
     state: started
+
+- name: Configure Cron tab centos
+  cron:
+    name: fstat {{item.name}}
```

Comment:
I think ansible will complain if that's not enclosed in " (I think, maybe this one is fine.